### PR TITLE
Update rest-client from 1.8.0 to 2.0.2

### PR DESCRIPTION
**Pull request by lukoktonos**
Update needed so that bonnie-qdm branch can update `rest-client` as part of updating to Ruby v2.4.5

See: https://github.com/projecttacoma/bonnie/pull/1170

Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1881
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: Zach Lister
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: health-data-standards.gemspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
 
   s.version = '3.7.0'
 
-  s.add_dependency 'rest-client', '~>1.8.0'
+  s.add_dependency 'rest-client', '~>2.0.1'
```
Comment (zlister) [inline health-data-standards.gemspec]: Need a space between `~>` and `2.0.1` for uniformity

**Review by zlister — APPROVED** [review]

**Review by mayerm94 — APPROVED** [review]
